system/Database/SQLite3/Table.php

```diff
@@ -200,25 +201,78 @@ public function modifyColumn(array $field)
      *
      * @return Table
      */
-    public function dropForeignKey(string $column)
+    public function dropForeignKey(string $foreignName)
     {
         if (empty($this->foreignKeys)) {
             return $this;
         }
```

Comment:
Changed the logic here to work with the new data structure of _foreignKeyData(). This was really ugly.

Interesting that this method was even here because there is no way the current implementation can even create foreign keys.